pavelib/servers.py

```diff
@@ -157,7 +157,7 @@ def celery(options):
     Runs Celery workers.
     """
     settings = getattr(options, 'settings', 'dev_with_worker')
-    run_process(django_cmd('lms', settings, 'celery', 'worker', '--beat', '--loglevel=INFO', '--pythonpath=.'))
+    run_process(django_cmd('studio', settings, 'celery', 'worker', '--beat', '--loglevel=INFO', '--pythonpath=.'))
```

Comment:
Do we need/want both of these lines?
How can/would the two sets of workers run simultaneously?

Reply:
I haven't looked into this much. The approach in edx docs _is_ scalable by running workers in separate terminals, but I wasn't able to spin up celery their way: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/environment/celery.html

Without knowing anything else on the subject, I think we would at least probably want separate commands / flags for specifying which worker to run rather than keeping both lines and spinning them both up.